module/VuFind/src/VuFind/Controller/AbstractSolrSearch.php

```diff
@@ -181,4 +204,27 @@ protected function getHierarchicalFacets($config)
             ? $facetConfig->SpecialFacets->hierarchical->toArray()
             : [];
     }
+
+    /**
+     * Get an array of hierarchical facet sort options
+     *
+     * @param string $config Name of facet configuration file to load.
+     *
+     * @return array
+     */
+    protected function getHierarchicalFacetsSortOptions($config)
```

Comment:
Should this be named in a way to clarify that this applies to advanced facets specifically?

Reply:
Renamed to getAdvancedHierarchicalFacetsSortOptions